src/styles.scss

```diff
@@ -0,0 +1,225 @@
+@import './vars';
+@import '~@primer/css';
+
+:root {
+  --primary-color: #384aff;
+  --bg-color: white;
+  --text-color: #586069;
+  --link-color: #111111;
+  --heading-color: #111111;
+  --border-color: #d1d5da;
+  --header-bg: white;
+  --block-bg: transparent;
+}
+
+@font-face {
+  font-family: 'Calibre-Medium';
+  src: url('./assets/fonts/Calibre-Medium.eot');
+  src: url('./assets/fonts/Calibre-Medium.eot?#iefix')
+      format('embedded-opentype'),
```

Comment:
this whole file comes straight from the snapshot app - this is the downside here, if snapshot starts chagning this file (and the vars) AND the pregov app stays in an iframe, then styling might start to slowly change when compared with snapshot